dbux-code/src/projectViews/practiceView/ProjectNode.js

```diff
@@ -59,27 +59,24 @@ export default class ProjectNode extends BaseTreeViewNode {
 
   async deleteProject() {
     if (isStatusRunningType(this.status)) {
-      showInformationMessage('project is running now...');
+      await showInformationMessage('project is running now...');
     }
     else {
-      const confirmMessage = `Do you really want to delete project: ${this.project.name}`;
+      const confirmMessage = `Do you really want to delete the project: ${this.project.name}`;
       const btnConfig = {
         Ok: async () => {
           await runTaskWithProgressBar(async (progress/* , cancelToken */) => {
             progress.report({ message: 'deleting project folder...' });
-            // wait for progress bar to show
-            await sleep(100);
-            this.project.deleteProjectFolder();
+            await this.project.deleteProjectFolder();
             this.treeNodeProvider.refresh();
-            progress.report({ message: 'Done.' });
           }, {
             cancellable: false,
-            location: ProgressLocation.Notification,
             title: this.project.name,
           });
         }
       };
       await showInformationMessage(confirmMessage, btnConfig, { modal: true });
+      await showInformationMessage('Project has been deleted successfully.');
```

Comment:
@MiccWan This should go into the `Ok` function, no?

Reply:
I'm so stupid :P